### PR TITLE
Updates to bpt prices

### PR DESCRIPTION
**Pull request by viniabussafi**
## Thank you for contributing to Spellbook!

The [docs](docs/) directory has been implemented to answer as many questions as possible. Please take the time to reference each `.md` file within this directory to understand how to efficiently contribute & why the repo is designed as it is 🪄. Example questions to be answered:
- What does each property in the [model config block](docs/models/model_config_block.md) mean?
- What is the [CI test](docs/ci_test/ci_test_overview.md) attached to PRs and how can I best utilize it?
- What are the Spellbook [best practices](docs/general/best_practices.md)?

Please navigate through the [docs](docs/) directory to find as much info as you can.

**Note**: happy to take PRs to improve the docs, let us know!

### CODE REVIEW

**Comment by viniabussafi** [comment]
hey @jeff-dude! this PR is still a work in progress, but can you please help me get past this error with the test? I tried forcing a run of the bpt supply spell but it still doesn't work

**Comment by jeff-dude** [comment]
> hey @jeff-dude! this PR is still a work in progress, but can you please help me get past this error with the test? I tried forcing a run of the bpt supply spell but it still doesn't work

we're looking, could be side effect of new CI setup for piloting sub-projects in spellbook

**Comment by couralex6** [comment]
This doesn't seem to be an issue with the CI. I tried running this locally and got the same error:

`TrinoUserError(type=USER_ERROR, name=SCHEMA_NOT_FOUND, message="line 204:15: Schema 'dbt_alex_dune' does not exist", query_id=20240202_231709_01161_t6hgc)`

Looks like a model generated by the macro doesn't have a schema defined. Maybe @Hosuke would be able to help debug? This could be an issue with the macro setup.

File: models/balancer/base/balancer_v2_base_bpt_prices.sql [hunk 1]
```diff
@@ -1,161 +1,16 @@
+{% set blockchain = 'base' %}
+
 {{
     config(
-        schema = 'balancer_v2_base',
+        schema='balancer_v2_' + blockchain,
```
Comment (Hosuke) [inline models/balancer/base/balancer_v2_base_bpt_prices.sql:5]: I think we may need to hardcode the schema here.

Comment (Hosuke) [reply models/balancer/base/balancer_v2_base_bpt_prices.sql]: If this solved the problem, you can change schema in other sqls as well.

Comment (Hosuke) [reply models/balancer/base/balancer_v2_base_bpt_prices.sql]: Also force-push may override the fixing commits I tried to submit.
But I think the main issue would be the schema.

**Comment by Hosuke** [comment]
Based on my observation:
`*.bpt_prices` depend on `balancer.bpt_supply`, which is refreshed during `dbt run`.
https://github.com/duneanalytics/spellbook/actions/runs/7798019125/job/21265853645?pr=5262#step:11:45

Quick solution:
Submit a PR for `*.bpt_supply` first (as seems they have been fully built), or separate `bpt_prices` and `bpt_supply` in two different PRs.

**Comment by viniabussafi** [comment]
> Quick solution:
> Submit a PR for *.bpt_supply first (as seems they have been fully built), or separate bpt_prices and bpt_supply in two different PRs.

Will do, @Hosuke. I'm closing this PR and starting fresh in a new one, in which I'll fix every occurence of `schema='balancer_v2_' + blockchain,` and then I'll run a new PR for the changes I need on bpt prices.
Thanks for your help!

**Pull request closed without merging**